data/EMT.txt

```diff
@@ -1,19 +1,19 @@
 Gene	Rule
```

Comment:
This is the change to the EMT file from when we decided to remove the underscores. A solution to the underscore issue has not yet been implemented by myself or anyone working on the project, so for now I have kept the EMT file this way.